features/home/src/main/java/com/techcafe/todone/home/HomeController.kt

```diff
@@ -0,0 +1,34 @@
+package com.techcafe.todone.home
+
+import com.airbnb.epoxy.TypedEpoxyController
+import com.techcafe.todone.androidcomponent.spacing
+
+class HomeController : TypedEpoxyController<List<Pair<String, String>>>() {
+    override fun buildModels(data: List<Pair<String, String>>) {
+        data.forEachIndexed { index, pair ->
+            val (projectName, description) = pair
+
+            if (data.isEmpty()) {
+                empty {
+                    id("empty")
+                }

```

Comment:
まあここ全然あとになおすとか，if elseで書くとか色々やり方あると思うけど，dataが空だったときはここ通ってこれ以降は下に行かないように早期returnしとくといいかも〜

```suggestion
            if (data.isEmpty()) {
                empty {
                    id("empty")
                }
                return
            }
```

Reply:
早期リターン大事b

Reply:
return忘れてました！ありがとうございます！